README.md

```diff
@@ -2,11 +2,858 @@
+# Manual Instrumentation Guide
+
+In this guide we will be using a running example. In section one and two, we will create two spring web services using Spring Boot. We will then trace the requests between these services using OpenTelemetry. Finally, in section three we will explore tools in the opentelemetry-contrib-spring package which can improve this process.
+
+## Create two Spring Projects
+
+Using the [spring project initializer](https://start.spring.io/), we will create two spring projects.  Name one project FirstService and the other SecondService. Make sure to select maven, Spring Boot 2.3, Java, and add the spring-web dependency. After downloading the two projects include the OpenTelemetry dependencies and configuration listed below. 
+
+## Setup for Section 1 and Section 2
+
+Add the dependencies below to enable OpenTelemetry in FirstService and SecondService. The Jaeger and LoggerExporter packages are recommended for exporting traces but are not required for this section. As of May 2020, Jaeger, Zipkin, OTLP, and Logging exporters are supported by opentelemetry-java. Feel free to use whatever exporter you are most comfortable with. 
+
+### Maven
+ 
+#### OpenTelemetry
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-api</artifactId>
+	<version>0.2.0</version>
+</dependency>
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-sdk</artifactId>
+	<version>0.2.0</version>
+</dependency>	
+```
+
+#### LoggerExporter
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-exporters-logging</artifactId>
+	<version>0.2.0</version>
+</dependency>
+```
+
+#### JaegerExporter
+```xml
+<dependency>
+	<groupId>io.opentelemetry</groupId>
+	<artifactId>opentelemetry-exporters-jaeger</artifactId>
+	<version>0.2.0</version>
+</dependency>
+<dependency>
+	<groupId>io.grpc</groupId>
+	<artifactId>grpc-protobuf</artifactId>
+	<version>1.27.2</version>
+</dependency>
+<dependency>
+	<groupId>io.grpc</groupId>
+	<artifactId>grpc-netty</artifactId>
+	<version>1.27.2</version>
+</dependency>
+```
+
+### Gradle
+ 
+#### OpenTelemetry
+```gradle
+compile "io.opentelemetry:opentelemetry-api:0.2.0"
+compile "io.opentelemetry:opentelemetry-sdk:0.2.0"
+```
+
+#### LoggerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-logging:0.2.0"
+```
+
+#### JaegerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-jaeger:0.2.0"
+compile "io.grpc:grpc-protobuf:1.27.2"
+compile "io.grpc:grpc-netty:1.27.2"
+```
+
+### Tracer Configuration
+
+To enable tracing in your OpenTelemetry project configure a TracerBean. This bean will be autowired to controllers to create and propagate spans. If you plan to use a trace exporter remember to also include it in this configuration file. In [section 3](#section-3-instrumentation-using-opentelemetry-contrib-spring-in-progress) we will provide an [annotation](#configtracer) to provide this functionality.
```

Comment:
We'll likely submit section 1 first, so the section 3 reference may end up being a bad link.

Also, don't have your shortlinks explicitly include the section numbering.

Reply:
I'll make this change in the official PR. I'll keep this change for future code reviews.